pkg/varlinkapi/images.go

```diff
@@ -665,7 +672,7 @@ func (i *LibpodAPI) PullImage(call iopodman.VarlinkCall, name string) error {
 			}
 		}
 		c <- nil
-		close(c)
+		defer close(c)
```

Comment:
Why?

Reply:
ensure the channel doesn't close prematurely